### DISCUSSION TITLE
Stuck in Creating

### DISCUSSION THREAD

**Discussion by fulcus** [community/community#55960] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Upon creation the website gets stuck as shown below

<img width="754" alt="Screenshot 2023-05-20 at 02 52 30" src="https://github.com/community/community/assets/33703910/266d4b21-9045-4bd1-88ee-9d6cb67e5fb3">

**Comment by hardal7** [comment]
Hi fuclus, have you tried creating a list from Create list button at top?

**Reply by SilenceG678** [reply]
No

**Comment by Uidc1000** [comment]
> Hi fuclus, have you tried creating a list from Create list button at top?

https://github.com/raulriera/TextFieldEffects/commit/b1b7185e0c1eb469084b8b033f0f0b4e679880ad

**Comment by takanome-dev** [comment]
The same happens to me every time I create a new list. It got stuck with `creating...` and won't close the modal. If you refresh the page, you'll see that the list has been created

**Reply by takanome-dev** [reply]
There is already a discussion about this -> https://github.com/orgs/community/discussions/51296#discussion-5015842

**Comment by Shpaga1** [comment]
1

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.

**Comment by rovo79** [comment]
Duplicate of #51296
